Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    langchainrb (0.3.13)
+    langchainrb (0.3.10)
```

Comment:
I think this change was accidental?

Reply:
I noticed there was another 0.3.13 -> 0.3.10 in this same PR. Make sure you fix them both @technicalpickles.

Thanks for doing this work!

Reply:
Not sure what happened here. Fixed it in any event.